source/manage/logging.rst

```diff
@@ -0,0 +1,487 @@
+
+        The separate configuration file includes the multi-line JSON instead.
+
+        In the example below, the first output is written to the console in plain text and includes all audit log levels, events, and command outputs. A pipe ``|`` delimiter is placed between fields. 
+
+        A second output is written to ``/logs/audit.log`` in plain text in a machine-readable JSON format and includes all audit log levels, events, and command outputs. Older logs are kept for 1 day, and up to a total of 10 backup log files are kept at a time. Logs are rotated using GZIP when the maximum size of the log file reaches 500 MB. A maximum of 1000 audit records can be queued/buffered while writing to the file.
+
+        Contents of ``audit_log_config.json`` file:
+
+        .. code-block:: JSON
+
+            {
+            "sample-console": {
+                "type": "console",
+                "format": "plain",
+                "format_options": {
+                    "delim": " | "
+                },
+                "levels": [
+                { "id": 100, "name": "audit-api" },
+                { "id": 101, "name": "audit-content" },
+                { "id": 102, "name": "audit-permissions" },
+                { "id": 103, "name": "audit-cli" }
+                ],
+                "options": {
+                "out": "stdout"
+                },
+                "maxqueuesize": 1000
+            },
+            "sample-file": {
+                "type": "file",
+                "format": "json",
+                "levels": [
+                { "id": 100, "name": "audit-api" },
+                { "id": 101, "name": "audit-content" },
+                { "id": 102, "name": "audit-permissions" },
+                { "id": 103, "name": "audit-cli" }
+                ],
+                "options": {
+                "compress": true,
+                "filename": "audit.log",
+                "max_age": 1,
+                "max_backups": 10,
+                "max_size": 500
+                },
+                "maxqueuesize": 1000
+            }
+            }
+
+Specify destination targets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Log records can be sent to any combination of `console <#console-target-configuration-options>`__, `local file <#file-target-configuration-options>`__, `syslog <#syslog-target-configuration-options>`__, and `TCP socket <#tcp-target-configuration-options>`__ targets. Log targets have been chosen based on support for the vast majority of log aggregators and other log analysis tools, without needing additional software installed.
+
+System Admins can define multiple log targets to:
+
+- Mirror log output to files and log aggregators for redundancy.
+- Log certain entries to specific destinations. For example, all ``audit-content`` records can be routed to a different destination than the other levels.
+- Admins can also temporarily disable log targets by setting its ``type`` to ``none``.
+
+Configure format preferences
```

Comment:
Should we have examples of how to use these? I see we have it in the tabs above under `fileSpec`, but that's the only time on the page i see `format_options` used

Reply:
I would suggest we defer this.  We can create a ticket to circle back and add a link to a large sample containing more options.

Reply:
Will do. Logged https://mattermost.atlassian.net/browse/MM-56354 to capture this ask.